documentation/asciidoc/computers/compute-module/cmio-display.adoc

```diff
@@ -1,16 +1,16 @@
-== Attach the official 7-inch display
+== Attaching the Touch Display LCD panel
 
 Update your system software and firmware to the latest version before starting.
 Compute Modules mostly use the same process, but sometimes physical differences force changes for a particular model.
 
-=== Connect a display to DISP1
+=== Connect a display to DISP1/DSI1
```

Comment:
Do any of the boards call it DSI1? CM5IO does call it CAM/DISP1.

Reply:
Not the boards, but the software always refers to it as DSI, rarely DISP. Created a rod for our own backs there.

Reply:
It's a bit like the way that the silkscreen labels the HDMI ports as HDMI0 and HDMI1, but the software refers to them as HDMI-A-1 and HDMI-A-2 :wink: